middlewares/validators/discordactions.js

```diff
@@ -28,7 +28,22 @@ const validateMemberRoleBody = async (req, res, next) => {
   }
 };
 
+const validateUpdateUsersNicknameStatusBody = async (req, res, next) => {
+  const schema = Joi.object({
+    lastNicknameUpdate: Joi.alternatives().try(Joi.string().trim(), Joi.number().unit("milliseconds")).required(),
```

Comment:
lastNicknameUpdate field is allowed to be either a string or a number, but the unit("milliseconds") validator only works with numbers. This means that if the lastNicknameUpdate field is a string, the validation will fail.

Reply:
Yes, thank you for pointing it out. There's no need to use `string()` validation. The validation to check if the numbers are sent properly as ms will be handled automatically by `number().unit()`. Even if they are strings, using `number()`, the validation is applied after trying to parse the value to number.